pkg/controller/plan/adapter/ova/builder.go

```diff
@@ -440,6 +440,12 @@ func (r *Builder) Tasks(vmRef ref.Ref) (list []*plan.Task, err error) {
 	return
 }
 
+func (r *Builder) PreferenceName(vmRef ref.Ref, configMap *core.ConfigMap) (name string, err error) {
+	// The VM is built from configuration, we don't need the preference
+	err = liberr.New("preferences are not used by this provider")
```

Comment:
this builder should be similar to that of vSphere, no?

Reply:
i think you forgot about the previous comment regarding this one.. in OVA we always set to `unknown` the OS we read from the OVF, so it doesn't matter here.
Generally - yes, it should be like vsphere.

Reply:
ah yeah, I had Déjà-vu when I wrote it :) so let's add a comment?

Reply:
suggested a change to the comment